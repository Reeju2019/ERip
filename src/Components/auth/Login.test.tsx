import { render, screen } from '@testing-library/react'
import Login from './Login'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'

test('auth login', () => {
  render(<Login />)
  const linkElement = screen.getByText(/Modal body text goes here./i)
  expect(linkElement).toBeInTheDocument()
})

describe('App rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<Login/>)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
    
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<Login />)
    ReactDOM.unmountComponentAtNode(div)
  })
})
