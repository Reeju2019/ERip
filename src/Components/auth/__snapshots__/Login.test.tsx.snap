// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<div>
  <div
    className="modal"
  >
    <div
      className="modal-dialog"
    >
      <div
        className="modal-content"
      >
        <div
          className="modal-header"
        >
          <h5
            className="modal-title"
          >
            Modal title
          </h5>
          <button
            aria-label="Close"
            className="btn-close"
            data-bs-dismiss="modal"
            type="button"
          />
        </div>
        <div
          className="modal-body"
        >
          <p>
            Modal body text goes here.
          </p>
        </div>
        <div
          className="modal-footer"
        >
          <button
            className="btn btn-secondary"
            data-bs-dismiss="modal"
            type="button"
          >
            Close
          </button>
          <button
            className="btn btn-primary"
            type="button"
          >
            Save changes
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
